.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+_Before filing a new issue, please **provide the following information**._
+
+> I'm running:
+>
+> - **Parity client version**: 0.0.0
```

Comment:
And the UI version maybe?

Reply:
I found it somehow not easy to find (in the release or in the console..) but we can try and see.